typescript/sdk/package.json

```diff
@@ -1,41 +1,52 @@
 {
   "name": "@abacus-network/sdk",
   "description": "The official SDK for the Abacus Network",
-  "version": "0.2.3",
+  "version": "0.3.0-beta0",
   "dependencies": {
+    "@abacus-network/app": "0.3.0-beta0",
     "@abacus-network/celo-ethers-provider": "^0.1.0",
-    "@abacus-network/core": "^0.2.3",
-    "@abacus-network/utils": "^0.2.3",
+    "@abacus-network/core": "0.3.0-beta0",
+    "@abacus-network/utils": "0.3.0-beta0",
+    "@types/debug": "^4.1.7",
```

Comment:
is this a `devDependency`?

Reply:
Good question. You may remember I raised that a week or two ago but actually this is what we want. Including it here is the least worst way to get things to work well for SDK consumers. We need it because our own types use the `debug` package types since we accept custom loggers